priv/apps/itest/lib/in_flight_exit_client.ex

```diff
@@ -0,0 +1,271 @@
+defmodule Itest.InFlightExitClient do
+  @moduledoc """
+  """
+
+  alias Itest.Transactions.Currency
+  alias Itest.Transactions.Encoding
+  alias Itest.Transactions.PaymentType
+  alias Itest.Client
+  alias WatcherSecurityCriticalAPI.Connection, as: Watcher
+  alias WatcherSecurityCriticalAPI.Api.InFlightExit
+  alias WatcherSecurityCriticalAPI.Model.InFlightExitTxBytesBodySchema
+
+  import Itest.Poller, only: [wait_on_receipt_confirmed: 2]
+  require Logger
+
+  @ife_bond_size 37_000_000_000_000_000
+  @ife_gas 2_000_000
+  @ife_gas_price 1_000_000_000
+  @retry_count 120
+  @min_exit_period 1_000
+  @gas_process_exit 5_712_388
+  @gas_process_exit_price 1_000_000_000
+  @gas_add_exit_queue 800_000
+
+  @gas_piggyback 1_000_000
+  @piggyback_bond 28_000_000_000_000_000
```

Comment:
Are all these magic ETH numbers fixed in the contract or can we set them to anything we want?

Reply:
@chrishunt you can technically set them higher, but I don't believe you can set them any lower. Pulled these from the other elixir-omg tests.

Reply:
hmmm, be aware the bond size currently stated is the initial hardcoded bond size from the contract. However, it is possible for contract to update the bond size. Depends on how the tests are set, might have impact or not.

Reply:
Is there a way to read them @boolafish ? seems like it would be better to get them on the fly

Reply:
I think you can use the `startIFEBondSize` and `piggybackBondSize` functions from the exit game contract (https://github.com/omisego/plasma-contracts/blob/c0ac3f958b7a4a06410368705ca90a8f183ea3a2/plasma_framework/contracts/src/exits/payment/routers/PaymentInFlightExitRouter.sol#L313)